commands/pipe/index.js

```diff
@@ -228,7 +229,12 @@ module.exports = {
 				};
 			}
 			else {
-				const string = sb.Utils.wrapString(result.reply, this.staticData.resultCharacterLimit, {
+				let reply = result.reply;
+				if (context.params._parsed && result.parsed) {
```

Comment:
This might wrongly reply with the normal response when the result is actually a falsy object like `""`, `0`, `false`, `null`

Reply:
`result.parsed` shouldn't ever be falsy if the command implements the `parsed` field correctly (which should be a stringified json object).

Having said that, I added an explicit check, good catch, thanks!